### ISSUE TITLE
The card going out of screen on Android

### ISSUE THREAD

**Issue by adorobis** [mweimerskirch/lovelace-hacomfoairmqtt#2]
This is excellent work! Just installed it on my HA. Now as it is almost perfect, on my android phone the left side of the card is slightly outside of the screen (Samsung Galaxy S8), please check the screenshot.
![image](https://user-images.githubusercontent.com/19752098/106486381-dfb37080-64b1-11eb-9d6e-1cd590be4382.png)

**Comment by mweimerskirch** [comment]
Thank you for reporting this. I'll work on making the card responsive later this week.

**Comment by adorobis** [comment]
Any news on that? Also in Chrome on PC depending on window width it is going out of view:
![image](https://user-images.githubusercontent.com/19752098/132850978-47e3de04-c4fb-4482-be90-8969be36495e.png)

**Comment by PimDoos** [comment]
Same issue on desktop, on certain aspect ratios the outside temperatures are going off screen: 
![image](https://user-images.githubusercontent.com/17709721/166123810-530bfd6b-869b-441d-804d-1646cae79258.png)

Any updates on this?

**Comment by zanac** [comment]
Workaround: add card-mod and use:
    card_mod:
      style: |
        ha-card {
         padding-left: 50px;
        }

**Comment by adorobis** [comment]
Good starting point but now it sticks out to the right. Is there a style element to scale it down?
![image](https://github.com/mweimerskirch/lovelace-hacomfoairmqtt/assets/19752098/72e7a736-89d5-4641-a99b-a44a941b49d3)

**Comment by zanac** [comment]
> Good starting point but now it sticks out to the right. Is there a style element to scale it down? ![image](https://user-images.githubusercontent.com/19752098/248327747-72e7a736-89d5-4641-a99b-a44a941b49d3.png)

just use padding-right ;)
    card_mod:
      style: |
        ha-card {
         padding-left: 50px;
         padding-right: 50px;
        }

**Comment by adorobis** [comment]
Nope, already tried. The house image and other elements don't scale

**Comment by zanac** [comment]
> Nope, already tried. The house image and other elements don't scale

maybe you wrong syntax, i use it and it seems works.

**Comment by adorobis** [comment]
Try to make the browser window smaller and most likely you'll see what I see.
```
card_mod:
  style: |
    ha-card{
      padding-left: 50px;
      padding-right: 50px;

    }
```
![image](https://github.com/mweimerskirch/lovelace-hacomfoairmqtt/assets/19752098/3f1eda6a-a622-4709-8033-8a5c94a0f7ba)

**Comment by zanac** [comment]
try to force also width at 75%.... here you are syntax
    card_mod:
      style: |
        ha-card {
         padding-left: 50px;
         padding-right: 50px;
         width: 75%;
        }

**Comment by adorobis** [comment]
That makes the whole card smaller and the content is still not scaled down...
